src/wallet/wallet.cpp

```diff
@@ -3370,8 +3370,12 @@ bool CWallet::GetBudgetSystemCollateralTX(CWalletTx& tx, uint256 hash, CAmount a
     std::vector< CRecipient > vecSend;
     vecSend.push_back((CRecipient){scriptChange, amount, false});
 
-    CCoinControl *coinControl=NULL;
-    bool success = CreateTransaction(vecSend, tx, reservekey, nFeeRet, nChangePosRet, strFail, coinControl, true, ALL_COINS, fUseInstantSend);
+    CCoinControl coinControl;
+    coinControl.SetNull();
```

Comment:
The constructor already does this call (would be strange if it didn't).

Reply:
```suggestion
```

Reply:
Ahh yes, I believe that came from me trying to use a pointer with it